migrations/versions/0387_populate_letter_branding.py

```diff
@@ -0,0 +1,31 @@
+"""
+
+Revision ID: 0387_populate_letter_branding
+Revises: 0386_email_branding_alt_text
+Create Date: 2022-11-23 14:04:41.456302
+
+"""
+from alembic import op
+
+revision = '0387_populate_letter_branding'
+down_revision = '0386_email_branding_alt_text'
+
+
+def upgrade():
+    op.execute(
+        """
+        INSERT INTO letter_branding_to_organisation
+        (organisation_id, letter_branding_id)
+        (SELECT organisation_id, letter_branding_id FROM services
```

Comment:
Inconsequential for the number of rows we're dealing with, but `SELECT DISTINCT` could've been nice here.